services/sshd.nix

```diff
@@ -0,0 +1,12 @@
+{ config, lib, pkgs, ... }:
+{
+  services.openssh = {
+    enable = true;
+    passwordAuthentication = false;
```

Comment:
I believe everyone is using ssh keys?

Reply:
Yes, in fact we've always had `services.openssh.passwordAuthentication = false;` set (in `profiles/security.nix`).

Reply:
Ok. I would still keep it here as well because I am importing this file into the kexec image.

Reply:
Sounds good to me! Just wanted to point out that we are already banning password logins :)